test/reduce.jl

```diff
@@ -16,6 +17,7 @@
 @test Base.mapfoldl((x)-> x ⊻ true, |, [true false true false false]) == true
 @test Base.mapfoldl((x)-> x ⊻ true, |, false, [true false true false false]) == true
 
+@test foldr(+, Int64[]) == 0
```

Comment:
Perhaps add a comment to each of these tests referencing the relevant issue and pull request?